python/oop/SOLID/bird/Bird.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC, abstractmethod
+import BirdType
+
+class Bird(ABC):
+	def __init__(self, weight, colour, size, beakType, birdType):
```

Comment:
You can use type hints to make it easier to understand data type. [Reference](https://realpython.com/lessons/type-hinting/#:~:text=Type%20hinting%20is%20a%20formal,and%20introduced%20in%20Python%203.5.&text=The%20name%3A%20str%20syntax%20indicates,function%20will%20return%20a%20string.)